codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttMessageType.java

```diff
@@ -46,12 +62,10 @@ public int value() {
     }
 
     public static MqttMessageType valueOf(int type) {
-        for (MqttMessageType t : values()) {
-            if (t.value == type) {
-                return t;
-            }
+        if (type <= 0 || type >= VALUES.length) {
+            throw new IllegalArgumentException("unknown message type: " + type);
```

Comment:
Exception messages should start capitalised. `"Unknown …"`

Reply:
I will fix this, but I've just copied the original exception here